pymedphys/labs/jothyselvaraj/pinnacle_batch_export.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (C) 2019 Jothy Selvaraj
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version (the "AGPL-3.0+").
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License and the additional terms for more
+# details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# ADDITIONAL TERMS are also included as allowed by Section 7 of the GNU
+# Affero General Public License. These additional terms are Sections 1, 5,
+# 6, 7, 8, and 9 from the Apache License, Version 2.0 (the "Apache-2.0")
+# where all references to the definition "License" are instead defined to
+# mean the AGPL-3.0+.
+
+# You should have received a copy of the Apache-2.0 along with this
+# program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
+
+
+import os
+import re
+import shutil
+import tarfile
+
+import numpy as np
+
+import xlrd
```

Comment:
Do you need this dependency? This will be new for PyMedPhys.

Reply:
You mean xlrd?

Reply:
Especially given the warning notice on the front page:

https://github.com/python-excel/xlrd#xlrd

Reply:
Replace ReadPatNames definition with this one and replace import xlrd with
from openpyxl import load_workbook

# Reads patients' names from a excel sheet names should be in format Lastname_Firstname
        def ReadPatNames(self, FilePath, SheetName, NamesColLetter):
            wb = load_workbook(filename=FilePath)
            sheet = wb[SheetName]
            NameCols = sheet[NamesColLetter]
            NamesLst=[]
            for x in range(len(NameCols)):
                NamesLst.append(NameCols[x].value)
            return NamesLst